addons/survey/models/__init__.py

```diff
@@ -8,3 +8,4 @@
 from . import badge
 from . import challenge
 from . import res_partner
+from . import ir_http
```

Comment:
order :hammer:  :slightly_smiling_face:  !

As we're in master, we could also reorder everything while we're here.